client/src/components/generic/AlertsComponent/AlertsComponent.tsx

```diff
@@ -0,0 +1,122 @@
+// import DefaultSm from "../FigmaButtons/FigmaButton"
+
+type AlertsInputProp = {
+  message: string
+  variant?: "success" | "notification" | "error"
+}
+
+const AlertsComponent = ({ variant, message }: AlertsInputProp) => {
+  if (variant === "success") {
+    return (
+      <button className="flex h-12 w-full items-center justify-between rounded-lg bg-[#109D27] text-left uppercase text-white">
+        <div className="ml-10 mr-5">
+          <svg
```

Comment:
The svgs should already be in `client/src/assets`, and you can import using the syntax like: 

```ts
import LeftArrowIcon from "assets/icons/leftarrow.svg?react"
// then
 <LeftArrowIcon className="fill-white" />
```